src/include/hip/rpp_hip_common.hpp

```diff
@@ -822,30 +850,30 @@ __device__ __forceinline__ void rpp_hip_load24_pkd3_and_unpack_to_float24_pln3(f
     srcPtr_f24 = (d_float24 *)srcPtr;
 
     src_f24->x.x.x = srcPtr_f24->x.x.x;
-    src_f24->y.x.x = srcPtr_f24->x.x.y;
-    src_f24->z.x.x = srcPtr_f24->x.x.z;
     src_f24->x.x.y = srcPtr_f24->x.x.w;
-    src_f24->y.x.y = srcPtr_f24->x.y.x;
-    src_f24->z.x.y = srcPtr_f24->x.y.y;
     src_f24->x.x.z = srcPtr_f24->x.y.z;
-    src_f24->y.x.z = srcPtr_f24->x.y.w;
-
-    src_f24->z.x.z = srcPtr_f24->y.x.x;
     src_f24->x.x.w = srcPtr_f24->y.x.y;
-    src_f24->y.x.w = srcPtr_f24->y.x.z;
-    src_f24->z.x.w = srcPtr_f24->y.x.w;
     src_f24->x.y.x = srcPtr_f24->y.y.x;
-    src_f24->y.y.x = srcPtr_f24->y.y.y;
-    src_f24->z.y.x = srcPtr_f24->y.y.z;
     src_f24->x.y.y = srcPtr_f24->y.y.w;
+    src_f24->x.y.z = srcPtr_f24->z.x.z;
+    src_f24->x.y.w = srcPtr_f24->z.y.y;
 
+    src_f24->y.x.x = srcPtr_f24->x.x.y;
+    src_f24->y.x.y = srcPtr_f24->x.y.x;
+    src_f24->y.x.z = srcPtr_f24->x.y.w;
+    src_f24->y.x.w = srcPtr_f24->y.x.z;
+    src_f24->y.y.x = srcPtr_f24->y.y.y;
     src_f24->y.y.y = srcPtr_f24->z.x.x;
-    src_f24->z.y.y = srcPtr_f24->z.x.y;
-    src_f24->x.y.z = srcPtr_f24->z.x.z;
     src_f24->y.y.z = srcPtr_f24->z.x.w;
-    src_f24->z.y.z = srcPtr_f24->z.y.x;
-    src_f24->x.y.w = srcPtr_f24->z.y.y;
     src_f24->y.y.w = srcPtr_f24->z.y.z;
+
+    src_f24->z.x.x = srcPtr_f24->x.x.z;
```

Comment:
similar to the above comment, assign src_f24->z.x as a float4. please make a similar change for src_f24->z.y

Reply:
Done